src/components/calculator.js

```diff
@@ -1,39 +1,113 @@
-import React from 'react';
```

Comment:
- [ ] Kindly rename this file name to `Calculator.js` as suggested by the previous reviewer as your app does not render with this issue.

  ![image](https://user-images.githubusercontent.com/54848363/236236255-5c537648-4cba-447d-afb4-9582dc55ba0a.png)